edb/lib/cal.edgeql

```diff
@@ -266,14 +318,51 @@ cal::date_get(dt: cal::local_date, el: std::str) -> std::float64
                 detail => (
                     '{"hint":"Supported units: century, day, ' ||
                     'decade, dow, doy, isodow, isoyear, ' ||
-                    'millenium, month, quarter, seconds, week, year."}'
+                    'millennium, month, quarter, seconds, week, year."}'
                 )
             )
         END
     $$;
 };
 
 
+CREATE FUNCTION
+cal::normalize_relative_hours(dur: cal::relative_duration)
+  -> cal::relative_duration
+{
+    CREATE ANNOTATION std::description :=
+        'Convert 24-hour chunks into days.';
+    SET volatility := 'Immutable';
+    SET force_return_cast := true;
+    USING SQL FUNCTION 'justify_hours';
+};
+
+
+CREATE FUNCTION
+cal::normalize_relative_days(dur: cal::relative_duration)
```

Comment:
Should this be defined for `date_duration` too? Also, why are we adding these functions? Are they essential?